website/docs/r/log_search.html.markdown

```diff
@@ -73,7 +73,17 @@ The following arguments are supported:
 - `description` - (Optional) Description of the search.
 - `parent_id` - (Required) The identifier of the folder to create the log search in.
 - `query_string` - (Required) Log query to perform.
-- `query_parameters` - (Optional) TODO Find a good description.
+- `query_parameters` - (Optional) Array of objects (LogSearchQueryParameterSyncDefinitionBase). Values for search template used in the search query. 
```

Comment:
Have you tested that this works? This is my first time looking at the code, but it seems like the argument name is `query_paramenter`, not `query_parameters`.
https://github.com/SumoLogic/terraform-provider-sumologic/blob/master/sumologic/resource_sumologic_log_search.go#L67

Reply:
Could we also add an example in the examples above?

Reply:
I think this would be similar to how we document `schedule` or `time_range`. In other words, it's a block list (with a max of 10, according to the docs you linked, but not enforced in the provider code 😞) where we can define multiple `query_parameter` blocks, e.g.

```
query_parameter {
  name        = "my_param1"
  description = "I am a query parameter. Fill me in."
  dataType    = "ANY"
  value       = "*"
}

query_parameter {
  name        = "my_param2"
  description = "I am also query parameter. Fill me in too."
  dataType    = "ANY"
  value       = "*"
}
```

I'm also not sure how `value` is supposed to support the same use case we see in the docs and UI ("Set Values for Parameter").